distributed/utils.py

```diff
@@ -359,12 +359,17 @@ def convert(task):
 
 import logging
 from .compatibility import logging_names
-logging.basicConfig(format='%(name)s - %(levelname)s - %(message)s',
-                    level=logging.INFO)
```

Comment:
Oh well.  Hrm, perhaps this is intended to be global and you're expected to just overwrite other people?  Perhaps call basicConfig yourself after importing `dask.distributed`?

My next step here would be to ask on stackoverflow what the right way to do this would be.